src/renderer/components/ft-community-post/ft-community-post.vue

```diff
@@ -8,16 +8,39 @@
     <div
       class="author-div"
     >
+      <router-link
+        v-if="authorThumbnails.length > 0 && authorId"
+        :to="`/channel/${authorId}`"
+        tabindex="-1"
+        aria-hidden="true"
+      >
+        <img
+          :src="getBestQualityImage(authorThumbnails)"
+          class="communityThumbnail"
+          alt=""
+        >
+      </router-link>
       <img
-        v-if="authorThumbnails.length > 0"
+        v-else-if="authorThumbnails.length > 0"
```

Comment:
Instead of checking the length of authorThumbnails in both the `v-if` and `v-else-if`, you could have a template with a `v-if` that wraps both and then a `v-if` authorId and `v-else` inside.